examples/sandbox/package.json

```diff
@@ -16,7 +16,7 @@
     "@shapeshiftoss/hdwallet-ledger-webusb": "^0.7.0",
     "@shapeshiftoss/hdwallet-trezor": "^0.7.0",
     "@shapeshiftoss/hdwallet-trezor-connect": "^0.7.0",
-    "@shapeshiftoss/hdwallet-portis": "^0.3.1",
+    "@shapeshiftoss/hdwallet-portis": "^0.3.2",
```

Comment:
don't bump the version manually. let `yarn lerna:version` do that once we've merged it

Reply:
(the CI scripts need the version bump and tagging to happen together)